core/src/main/java/org/openedx/core/ui/ComposeCommon.kt

```diff
@@ -163,9 +163,10 @@ fun Toolbar(
 
         Text(
             modifier = Modifier
+                .fillMaxWidth()
                 .testTag("txt_toolbar_title")
-                .align(Alignment.CenterVertically)
-                .padding(end = 16.dp),
+                .align(Alignment.Center)
```

Comment:
For Android, the toolbar title is supposed to be left aligned.

@moiz994 thoughts?

Reply:
> For Android, the toolbar title is supposed to be left aligned.
> 
> @moiz994 thoughts?

It's not entirely right. The Toolbar title can actually be aligned to any side. Most screens in the Open edX apps have a toolbar aligned to the center, as it requires our design system, but only a few mentioned above show with wrong alignments. 
Also, here are some links to the docs that show a center-aligned toolbar:
https://m3.material.io/components/top-app-bar/overview
https://developer.android.com/jetpack/compose/components/app-bars#top-bar

If the edX design doesn't allow a center-aligned toolbar, you have the option to implement it differently, either as part of theming or within an edX fork.

Reply:
> For Android, the toolbar title is supposed to be left aligned.

@ekangedx @sdaitzman thoughts?

Reply:
> > For Android, the toolbar title is supposed to be left aligned.
> 
> @ekangedx @sdaitzman thoughts?

@farhan-arshad-dev, we already have center-aligned titles all over the app, so starting a discussion on this isn't necessary. 
This PR just fixes the inconsistency in those three screens. 
Let's wrap up the review ASAP without dragging it out 🙏 
If we want to tweak the design system, we can discuss it during Tuesday's Mobile Design Weekly meeting.